homeassistant/components/circuit/notify.py

```diff
@@ -0,0 +1,38 @@
+"""Unify Circuit platform for notify component."""
+import logging
+
+from circuit_webhook import Circuit
+
+from homeassistant.components.notify import ATTR_TARGET, BaseNotificationService
+from homeassistant.const import CONF_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the Unify Circuit notification service."""
+    if discovery_info is None:
+        return
+
+    return CircuitNotificationService(discovery_info)
+
+
+class CircuitNotificationService(BaseNotificationService):
+    """Implement the notification service for Unify Circuit."""
+
+    def __init__(self, webhook_url):
```

Comment:
`config` is a better name since this is a dict with more than one item.

Reply:
Ok, will change this.